src/DETHRACE/common/loading.c

```diff
@@ -1795,7 +1795,11 @@ void LoadCar(char* pCar_name, tDriver pDriver, tCar_spec* pCar_spec, int pOwner,
         FatalError(kFatalError_FileCorrupt_S, pCar_name);
     }
     if (*pDriver_name != '\0') {
+#if defined(DETHRACE_FIX_BUGS)
```

Comment:
could you add a comment documenting what the bug is that we are fixing?

Reply:
The OG game (carm95.exe at least) does not do this.
It keeps the original key and does not decode the comments with the alternative key.
I'll add this as a comment.